### ISSUE TITLE
a way to have a rate my experience question 

### ISSUE THREAD

**Issue by shaba721** [Dragory/modmailbot#342]
maybe an option for a close message is to have a rate my experince with either words or numbers and then once they send it actually closes the ticket.

**Comment by Dragory** [comment]
Good idea! The option could add the reactions [1] [2] [3] [4] [5] to the close message or something. This would also naturally have to include some sort of a way to track the avg ratings and similar stats somewhere.

**Comment by shaba721** [comment]
you could do a score board with each number showing how many rated each got or possibly have it show an average score for whom ever closed the ticket

**Comment by ANONYM-ANONYM** [comment]
Great idea 👍 Please add it
